ouroboros-consensus-diffusion/src/ouroboros-consensus-diffusion/Ouroboros/Consensus/Node.hs

```diff
@@ -401,6 +429,7 @@ runWith :: forall m addrNTN addrNTC versionDataNTN versionDataNTC blk p2p.
      , Hashable addrNTN -- the constraint comes from `initNodeKernel`
      , NetworkIO m
      , NetworkAddr addrNTN
+     , MonadBase m m
```

Comment:
Is this noisy `MonadBase m m` constraint avoidable?

Reply:
https://github.com/IntersectMBO/ouroboros-consensus/pull/1267#discussion_r1836813794

Reply:
Moved into `IOLike`.